tests/test_post_with_faker.py

```diff
@@ -5,6 +5,10 @@
 
 fake = Faker()
 
+def test_get_product():
```

Comment:
can you please update the function name to something like "test_get_all_products"

Reply:
function name renamed, thanks Indu.